packages/components/organisms/f-checkout/src/components/_tests/ErrorDialog.test.js

```diff
@@ -13,7 +13,7 @@ const localVue = createLocalVue();
 localVue.use(VueI18n);
 localVue.use(Vuex);
 
-describe('ErrorDialog', () => {
+xdescribe('ErrorDialog', () => {
```

Comment:
Guessing this should be removed before it's merged?

Reply:
Hey, yeah it's just a very rough draft at the moment. had to ignore these to commit to github :)